docs/sphinx_extensions.py

```diff
@@ -167,3 +211,7 @@ def setup(app: Sphinx) -> None:
         app (Sphinx): The Sphinx application instance.
     """
     app.add_directive("exec-literalinclude", ExecLiteralInclude)
+    app.add_node(
+        ErrorMessageNode, html=(visit_error_message_node, depart_error_message_node)
```

Comment:
This can be replaced with

```suggestion
        ErrorMessageNode, html=(visit_error_message_node, None)
```

Then the `depart_error_message_node` can be removed

Reply:
Tried this and it does not work, it requires a function like that one.

Reply:
Okay 👍
Then the `self` in the `depart_error_message_node ` should also be renamed to `visitor`